### ISSUE TITLE
metric depth or relative depth ?

### ISSUE THREAD

**Issue by YacineDeghaies** [RuijieZhu94/mmdepth#2]
I want to customize your model on my own dataset, which has rgb images + relative depth maps( 8-bit)

Is this possible ?

One example of my ground truth depth maps:
![gt](https://github.com/user-attachments/assets/31ad9095-749c-4cce-b499-cad3debc5233)

**Comment by RuijieZhu94** [comment]
Sure. You need to abandon the scale prediction branch and change the depth loss to scale-invariant.

**Comment by YacineDeghaies** [comment]
Since i'm new to the field, can you give me some tips on which files should I start investigating ? 
should I implement a scale-invariant loss function and add it to the code ?

Your repository is very good documented. Thank you !

**Comment by RuijieZhu94** [comment]
Set `with_scale=False` in the [config file](https://github.com/RuijieZhu94/mmdepth/blob/main/projects/ScaleDepth/configs/ScaleDepth/scaledepth_clip_KITTI_352x1120.py), and change the [sigloss](https://github.com/RuijieZhu94/mmdepth/blob/main/projects/ScaleDepth/loss/sigloss.py) to scale-invariant loss. Also, don't forget to customize your own [dataloader](https://github.com/RuijieZhu94/mmdepth/blob/main/mmdepth/datasets/basesegdataset.py).

**Comment by RuijieZhu94** [comment]
Closing this issue. If there are any other questions please open another issue.
